src/runtime_src/tools/xclbinutil/XclBinUtilities.cxx

```diff
@@ -86,6 +85,10 @@ void XclBinUtilities::QUIET(const std::string &_msg) {
   }
 }
 
+void XclBinUtilities::QUIET(const boost::format& fmt) {
+  QUIET(boost::str(fmt));
```

Comment:
```suggestion
  QUIET(fmt.str());
```
Any reason for not using this? Just curious

Reply:
Having two QUIET signatures helps the user write clean code.  In this case, QUIET has the followign two signatures:
1. QUIET(const std::string &)
2. QUIET(const boost::format &)

Original, there was just the first.  But it became clumbersome to use and read.  For example:
* QUIET(boost::str(boost::format("This is a sample output file: '%s'") % std::string("myfile")));
* QUIET((boost::format("This is a sample output file: '%s'") % std::string("myfile")).str());
* QUIET(std::string("This is a sample output file: '") + "myfile");

Having the second signature, make the code easier to work with.  

Ultimately, we wish all the QUIET bodies to use a common one that will do the work.  This reduces duplicate code and makes it easier to support as well.